src/core/entity.js

```diff
@@ -55,44 +59,48 @@ class Entity {
     }
   }
 
-  look_at(vec, up = this.up) {
-    math.mat4.target_to(this.local_matrix, this.position, vec, up);
-  }
-
-  get left() {
```

Comment:
This wasn't used anywhere. `forward` and `up` are enough for most of the cases. It's also easy to compute if needed.